### PR TITLE
Add landing page

### PR DESCRIPTION
**Pull request by ramyabuva**
I moved the previous index files, and changed the naming convention to "scheduler". The new index page is an info page: 
(https://screenshot.googleplex.com/vEUASa5y0bk.png). I also added a favicon.

### CODE REVIEW

**Review by savsa — APPROVED** [review]

File: src/main/webapp/vendor/startbootstrap/template.jsp [hunk 1]
```diff
@@ -131,7 +132,7 @@
         <footer class="sticky-footer bg-white">
           <div class="container my-auto">
             <div class="copyright text-center my-auto">
-              <span>Copyright &copy; Capstone-Filler</span>
+              <span>Copyright &copy; scheduli</span>
```
Comment (wli) [inline src/main/webapp/vendor/startbootstrap/template.jsp]: this actually needs to be "Google LLC", since scheduli isn't a registered entity of any sort.

**Review by wli — CHANGES_REQUESTED** [review]
Where did the favicon image come from? Is that also from font awesome?

**Comment by ramyabuva** [comment]
> Where did the favicon image come from? Is that also from font awesome?

Yeah, the favicon came from font awesome too.

File: src/main/webapp/vendor/startbootstrap/template.jsp [hunk 2]
```diff
@@ -131,7 +132,7 @@
         <footer class="sticky-footer bg-white">
           <div class="container my-auto">
             <div class="copyright text-center my-auto">
-              <span>Copyright &copy; Capstone-Filler</span>
+              <span>Copyright &copy; Google LLC</span>
```
Comment (wli) [inline src/main/webapp/vendor/startbootstrap/template.jsp]: one more nit - let's add the year like we do in all the source files too.

**Review by wli — APPROVED** [review]

**Review by triplehair — APPROVED** [review]

**Pull request merged**